scripts/process_from_txt_to_list_tracknet.py

```diff
@@ -94,11 +91,8 @@ def process_input_data(file_name,
         chunk_data_moment = np.stack(chunk_data_moment, axis=0)
         chunk_data_real = np.stack(chunk_data_real, axis=0)
         chunk_data_len = np.stack(chunk_data_len, axis=0)
-        chunk_data_event = np.ones(chunk_data_len.shape)
-        chunk_data_event *= id
-
-        chunk_data_event_last_station = np.ones(last_station.shape[0])
-        chunk_data_event_last_station *= id
+        chunk_data_event = id
```

Comment:
if `chunk_data_event` and `chunk_data_event_last_station` is always the same, why do we need both of them?

Reply:
they have different shape

Reply:
number of last station hits doesn't  depend on number of trackcandidates at all